app/actions/v3/service_binding_create.rb

```diff
@@ -76,6 +96,20 @@ def save_last_operation(binding, details)
         )
       end
 
+      def save_incomplete_binding(binding, broker_operation)
+        binding.save_with_attributes_and_new_operation(
+          {},
+          {
+            type: 'create',
+            state: 'in progress',
+            broker_provided_operation: broker_operation
+          }
+        )
+        event_repository.record_start_create(binding, @user_audit_info, @audit_hash, manifest_triggered: false)
```

Comment:
nit: not manifest_triggered is by default false so no need to define it here (and in complete_binding_and_save )